### PR TITLE
Unity docs changes

### PR DESCRIPTION
**Pull request by ndesai-newrelic**

### CODE REVIEW

File: src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+</Step>
+
+<Step>
+### (iOS-only) Manage dependencies [#manage-dependencies]
+
+For iOS, check your External Dependency Manager settings:
+
+1. In your Unity IDE, go to **Assets** and open the **External Dependency Manager**.
+2. Click **iOS Resolver**.
+3. Settings: [#configure-ios]
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx:168]: @ndesai-newrelic Is this supposed to link to content somewhere?

File: src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+        dependencies {
+            // If you are changing the Android Gradle Plugin version, make sure it is compatible with the Gradle version preinstalled with Unity.
+            // See which Gradle version is preinstalled with Unity here: https://docs.unity3d.com/Manual/android-gradle-overview.html
+            // See the official Gradle and Android Gradle Plugin compatibility table here: https://developer.android.com/studio/releases/gradle-plugin#updating-gradle
+            // To specify a custom Gradle version in Unity, go to "Preferences > External Tools", uncheck "Gradle Installed with Unity (recommended)", and specify a path to a custom Gradle version.
+            classpath 'com.newrelic.agent.android:agent-gradle-plugin:7.2.0'
+            **BUILD_SCRIPT_DEPS**
+        }
+    }
+
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx:131]:
```suggestion
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx (hunk 1 above)
Comment (ndesai-newrelic) [reply src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx:168]: @ally-sassman no.there is no link for that

Comment (ally-sassman) [reply src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx:168]: What's the [#configure-ios] for then?

Comment (ndesai-newrelic) [reply src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx:168]: i think it is not needed.

File: src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+1. In your Unity IDE, go to **Assets** and open the **External Dependency Manager**.
+2. Click **iOS Resolver**.
+3. Settings: [#configure-ios]
+
+Within the Podfile Configurations section, please verify the following settings:
+
+* "Add use_frameworks! to Podfile" should be unchecked.
+* "Always add the main target to Podfile" should be checked.
+
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx:174]:
```suggestion
3. In the Podfile Configurations section, please verify the following settings:

* "Add use_frameworks! to Podfile" should be unchecked.
* "Always add the main target to Podfile" should be checked.

```

@ndesai-newrelic Like this? ^

**Review by ghost — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**